fossunited/www/grants/grantees/rss.py

```
@@ -0,0 +1,137 @@
+                "author": escape_html(g.event_organiser or ""),
+                "published_date": format_datetime(frappe.utils.get_datetime(date)),
+                "modified": g.modified,
+                "category": "Event",
+                "content": (
+                    f"<![CDATA["
+                    f"<p><strong>Grant Type:</strong> Event Grant</p>"
+                    f"<p><strong>Organiser:</strong> {escape_html(g.event_organiser or '')}</p>"
+                    f"<p><strong>Amount:</strong> {escape_html(amount)}</p>"
+                    f"{description}"
+                    + (f'<p><a href="{url}">Event Website →</a></p>' if url else "")
+                    + f"]]>"
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)